blog/content/post/2021-11-09-3mdeb-contribution-q3.md

```diff
@@ -0,0 +1,168 @@
+---
+title: 3mdeb contribution 2021'Q3
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/image-file.png
+author: daria.zakrzewska
+layout: post
+published: true
+date: 2021-11-09
+archives: "2021"
+
+tags:
+  - contribution
+  - patches
+  - coreboot
+  - opensource
+categories:
+  - Miscellaneous
+
+---
+## Intro 
+
+We are back with a post about corporate contributions, now focusing on the third
+quarter of 2021. It is usually said that work during summer is more difficult
+because of the beautiful weather outside, which does not allow you to focus on
+tasks. With us, however, it is possible to combine everything that is important,
+without neglecting the sense of the obligation to contributing in open source
+communities. The third quarter was active and we are so glad to take part in
+such important projects. What exactly? We invite you to read below, where you
+will find all the details! 
+
+But let’s start with a small riddle! 
+
+If you change, add, multiple some numbers in `2021’Q3` what will you get?
+ 
+## 2180 
+
+YES! all added lines in all project! Remember, not the quantity but
+quality is what we care about!
+
+1. Trenchboot Landing Zone
+
+Open source implementation of Secure Loader for AMD Secure Startup.
+
+Author | Category | Patch | URL
+Krystian Hebel | ?iommu | initial implementation| (LINK)[https://github.com/TrenchBoot/landing-zone/pull/63]
+Krystian Hebel | Build | Event log | (LINK)[https://github.com/TrenchBoot/landing-zone/pull/65]
+Krystian Hebel | Security | fix order of outb() arguments | (LINK)[https://github.com/TrenchBoot/landing-zone/pull/66]
+Krystian Hebel | README | Adding Multiboot2 support | (LINK)[https://github.com/TrenchBoot/landing-zone/pull/64]
+Krystian Hebel | Main | Headers Redesign | (LINK)[https://github.com/TrenchBoot/landing-zone/pull/68]
+Krystian Hebel | ??? | Xen support | (LINK)[https://github.com/TrenchBoot/landing-zone/pull/60]
+
+1. Freebsd 
```

Comment:
@DariaZakrzewska I'm afraid this useful feature of markdown won't work with tables between, so let's use default numbered lists